PublishHtmlReport/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.2.1",
   "description": "Pipelines task that publishes HTML report files",
   "author": "Jakub Rumpca, BlakYaks",
-  "main": "index.js",
+  "main": "index.ts",
   "private": true,
   "dependencies": {
-    "azure-pipelines-task-lib": "^4.1.0",
+    "azure-pipelines-task-lib": "^4.17.3",
```

Comment:
Update in an attempt to fix some of the dependency warnings, didn't fix them, but as far as I can tell, didn't cause any issues.